runtime/syntax/raku.yaml

```
@@ -1,7 +1,7 @@
 filetype: raku
 
 detect:
-    filename: "(\\.p6$|\\.pl6$|\\.pm6$|\\.raku$|\\.rakumod$|\\.rakudoc$)"
+    filename: "(\\.p6$|\\.pl6$|\\.pm6$|\\.raku(mod|doc|test)?$|\\.nqp$)"
```

```suggestion
    filename: "(\\.p(l|m|od)?6$|\\.raku(mod|doc|test)?$|\\.nqp$)"
```

Sorry for not mentioning it yesterday already, but the perl6 legacy extension can be merged too.